src/README.md

```diff
@@ -89,31 +92,61 @@ Like gRPC, RPC are serialized via protocol buffers.
+
+Headers for the official DAOS API exposed to the end user (i.e. I/O middleware or application developers) are under src/include and use the daos\_ prefix. Each infrastructure library exports an API that is available under src/include/daos and can be used by any services. The client-side API (with dc\_ prefix) exported by a given service is also stored under src/include/daos whereas the server-side interfaces (with ds\_ prefix) are under src/include/daos_srv.
 
 <a id="33"></a>
-### Infastructure & Microservices
-
-- <a href="vos/README.md">Versioning Object Store</a>
-- <a href="bio/README.md">Blob I/O</a>
-- <a href="rdb/README.md">Service Replication</a>
-- <a href="pool/README.md">Pool Service</a>
-- <a href="container/README.md">Container Service</a>
-- <a href="object/README.md">Key Array Object</a>
-- <a href="placement/README.md">Algorithmic Object Placement</a>
-- <a href="rebuild/README.md">Self-healing</a>
-- <a href="security/README.md">Security</a>
-- <a href="client/api/README.md">DAOS Client Library</a>
-- <a href="client/dfs/README.md">POSIX File & Directory Emulation</a>
+### Infrastructure Libraries
+
+The GURT and common DAOS (i.e. libdaos_common) libraries provide logging, debugging and common data structures (e.g. hash table, btree, ...) to the DAOS services.
+
+Local NVM storage is managed by the Versioning Object Store (VOS) and blob I/O (BIO) libraries. VOS implements the persistent index in SCM whereas BIO is responsible for storing application data in either NVMe SSD or SCM depending on the allocation strategy. The VEA layer is integrated into VOS and manages block allocation on NVMe SSDs.
+
+DAOS objects are distributed across multiple targets for both performance (i.e. sharding) and resilience (i.e. replication or erasure code). The placement library implements different algorithms (e.g. ring-based placement, jump consistent hash, ...) to generate the layout of an object from the list of targets and the object identifier.
+
+The replicated service (RSCV) library finally provides some common code to support fault tolerance. This is used by the pool, container & management services in conjunction with the RDB library that implements a replicated key-value store over Raft.
+
+For further reading on those infrastructure libraries, please see:
 - <a href="common/README.md">Common Library</a>
+- <a href="vos/README.md">Versioning Object Store (VOS)</a>
+- <a href="bio/README.md">Blob I/O (BIO)</a>
+- <a href="placement/README.md">Algorithmic object placement</a>
+- <a href="rdb/README.md">Replicated database (RDB)</a>
+- <a href="rsvc/README.md">Replicated service framework (RSVC)</a>
+
+<a id="34"></a>
+### DAOS Services
+
+The diagram below shows the internal layering of the DAOS services and interactions with the different libraries mentioned above.
+![DAOS Internal Layering](/doc/graph/layering.png)
```

Comment:
Looks like right now the diagram only shows dRPC communications going from the data plane to control plane for security. The flow for manipulating ACLs is: dmg -> control plane (via gRPC) -> data plane (via dRPC)

Reply:
right, that was for the signature verification. Will fix that.